docs/reference/run.md

```diff
@@ -900,6 +901,11 @@ If you do block IO in the two containers at the same time, by, for example:
 You'll find that the proportion of time is the same as the proportion of blkio
 weights of the two containers.
 
+The `--blkio-weight-device` flag can set the weight of a particular device to a value between 10 to 1000. With this, override the default value of group weight as specified by blkio.weight.
+For example, the command below create one container and set weight of /dev/sda 
+different blkio weight:
+    $ docker run -ti --name c3 --blkio-weight 300 --blkio-weight-device "/dev/sda:200" ubuntu:14.04 /bin/bash
```

Comment:
can you add a blank line before the code example?
also; could you wrap it? That would prevent a horizontal scrollbar, e.g.

```bash
$ docker run -ti \
    --blkio-weight 300 \
    --blkio-weight-device "/dev/sda:200" \
    ubuntu:14.04 /bin/bash
```